docs/java-rest/high-level/indices/put_settings.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[java-rest-high-indices-put-settings]]
+=== Update Indices Settings API
```

Comment:
Shall we call the page "Update Settings API" given that it's under the Indices API group already?

Reply:
:+1: I would prefer to shorten it  

But https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html
is using `Update Indices Settings` and currently the high level rest client docs are using the same names as the ones in the reference docs...

To keep the consistency between the client and the ref docs, maybe the ref docs should be changed as well?

@javanna what do you think?

Reply:
leave it as-is then :)